### DISCUSSION TITLE
Ablation experiments about Deeplabv3plus

### DISCUSSION THREAD

**Discussion by duduzai2019** [open-mmlab/mmsegmentation#2204] [General]
Hi in the ablation experiments remove the aspp and c1_output module in the mmseg/models/decode_heads/sep_aspp_head.py, but with little change, the miou maintained at around 80％,
What is wrong? has been troubing me for a long time!

**Comment by MengzhangLI** [comment]
Hi, from the [DeepLabV3+ paper](https://arxiv.org/pdf/1802.02611.pdf) figure it shows the `c1_output` (the yellow boxes below) contains low-level features which is very important for dense prediction tasks (such as semantic segmentation).

![image](https://user-images.githubusercontent.com/24582831/196337911-ba086421-320f-4f3c-9ff2-6ee26ae5cbd2.png)

**Comment by duduzai2019** [comment]
> Hi, from the [DeepLabV3+ paper](https://arxiv.org/pdf/1802.02611.pdf) figure it shows the `c1_output` (the yellow boxes below) contains low-level features which is very important for dense prediction tasks (such as semantic segmentation).
> 
> ![image](https://user-images.githubusercontent.com/24582831/196337911-ba086421-320f-4f3c-9ff2-6ee26ae5cbd2.png)

Thanks for your answer, i understand how it works, but i want to remove both modules in mmsegementation, Do i have a problem modifying it like this?  (mmseg/models/decode_heads/sep_aspp_head.py class DepthwiseSeparableASPPModule)
![BH$O3@FQ8U9PP2K9B19%S$J](https://user-images.githubusercontent.com/96464964/196349217-16872a5c-966a-47a5-98f7-4ec7ec09fd41.png)
Thanks again for your answer

**Comment by duduzai2019** [comment]
Hi, The mmsegementation effect is so good. i want to make the effect worse. Are there any suggestions?
